sdk/core/amqp-common/changelog.md

```diff
@@ -1,5 +1,8 @@
-### 2020-04-21 1.0.0-preview.13
+### 2020-04-01 1.0.0-preview.13
```

Comment:
lol, we are going back in time now
The date changed from 21st April to 1st April :)

Reply:
Damn!

Reply:
Might have been a result of merge conflict resolution.